### PR TITLE
Hw 4 5

### PR DESCRIPTION
**Pull request by darya-startsava**
Task: TODO list with Redux and React Router
Deploy: https://darya-startsava.github.io/todo-list
Screenshot: 
image

### CODE REVIEW

**Review by polina-ivanova — APPROVED** [review]
Great job and it works well in testing. I just don't understand why there are so many additional containers for many components. In my opinion, it makes the code harder to read. I would make a redux directory with store, all actions and all reducers.

File: src/components/home/home.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Form from '../form/container/form';
+import { labels } from '../../constants';
+
+export default function Home() {
+  return (
+    <>
```
Comment (poulkud) [inline src/components/home/home.jsx]: тут лишний

**Review by poulkud — APPROVED** [review]
Балдежно!

File: src/components/home/home.jsx (hunk 1 above)
Comment (darya-startsava) [reply src/components/home/home.jsx]: fixed

**Pull request merged**